api/sft/handler.go

```diff
@@ -27,11 +28,25 @@ func NewHandler(r chi.Router, s *Service) *Handler {
 
 func StartDashboard(port string) {
 
-	// TODO: sort out path to the dashboard
-	fs := http.FileServer(http.Dir("/Users/maxbb/github/revitt/sft/web/dashboard"))
-	http.Handle("/", fs)
+	//TODO: sort out path to the dashboard - this needs to be managed by a config when the dashboard is ready
+	buildPath := "/Users/maxbb/github/revitt/sft/web/dashboard/dist"
```

Comment:
Need to figure this out better - perhaps it should be an env var - this will differ based on container/calling app run context.

Reply:
@elliothllm this would be a good one for you to give some input on :)